analysis/tests/src/expected/TypeContextCompletion.res.txt

```diff
@@ -11,36 +11,184 @@ Found type for function (~someVariant: someVariant) => string
     "documentation": null
   }]
 
-Complete src/TypeContextCompletion.res 20:44
-posCursor:[20:44] posNoWhite:[20:43] Found expr:[20:11->20:44]
-Pexp_apply ...[20:11->20:30] (~someVariant20:32->20:43=...__ghost__[0:-1->0:-1])
+Complete src/TypeContextCompletion.res 21:44
+posCursor:[21:44] posNoWhite:[21:43] Found expr:[21:11->21:44]
+Pexp_apply ...[21:11->21:30] (~someVariant21:32->21:43=...__ghost__[0:-1->0:-1])
 found typed context 
-Completable: CtypedContext(Value[someVariantToString])
-[]
+Completable: CtypedContext(Value[someVariantToString], NamedArg(someVariant))
+Found variant type for NamedArg typed context someVariant
+[{
+    "label": "One",
+    "kind": 4,
+    "tags": [],
+    "detail": "One\n\n",
+    "documentation": null
+  }, {
+    "label": "Two",
+    "kind": 4,
+    "tags": [],
+    "detail": "Two\n\n",
+    "documentation": null
+  }, {
+    "label": "Three",
+    "kind": 4,
+    "tags": [],
+    "detail": "Three\n\n",
+    "documentation": null
+  }, {
+    "label": "Four",
+    "kind": 4,
+    "tags": [],
+    "detail": "Four\n\n",
+    "documentation": null
+  }, {
+    "label": "Five(_)",
+    "kind": 4,
+    "tags": [],
+    "detail": "Five(int)\n\n",
+    "documentation": null
+  }, {
+    "label": "Six(_)",
+    "kind": 4,
+    "tags": [],
+    "detail": "Six(option<string>)\n\n",
+    "documentation": null
+  }]
```

Comment:
Barring details like the payloads, filtering according to what the user has already typed, whether we should use snippets etc, this at least produces the output we're after - all of the variant constructors.

Reply:
Filtering is the next step. Nothing else need to be done.
Snippets etc we don't have to worry about. Perhaps later.